lib/maze/hooks/appium_hooks.rb

```diff
@@ -121,9 +121,9 @@ def device_capabilities(config, tunnel_id = nil)
                                                               config.os,
                                                               config.os_version,
                                                               config.capabilities_option
-          capabilities['bitbar_app'] = config.app
-          capabilities['bundleId'] = config.app_bundle_id
```

Comment:
Deliberately removed?

Reply:
Good point, actually, it may have been the cause for the issue on iOS.  I'm discussing with out Bitbar colleagues now.

Reply:
Having discussed it, Bitbar are changing their software so that the bundle Id will not be needed on iOS (great news for us).  That should be realised within the next couple of weeks, so I'm happy to merge this as-is now.